pkg/console/subresource/configmap/brand_ocp.go

```diff
@@ -0,0 +1,7 @@
+// +build ocp
+
+package configmap
+
+const (
+	BRAND = "ocp"
```

Comment:
Got me curious about the default docs URL as well... if that should be toggled for `ocp` vs `okd`.  Currently in `configmap.go` its just `"https://docs.okd.io/4.0/"`.

Reply:
@ahardin-rh helped dig up this alternative URL: https://docs.openshift.com/container-platform/4.0/welcome/index.html

I don't know yet if we should use or not (it is also password protected).  Fishing around for this.

Reply:
@zherman0 yes, I believe this is the correct URL to use. It is password protected for now as it is under active dev. It will not be later.